tests/acceptance/PreviewPostNotificationNewsletterCest.php

```diff
@@ -41,26 +41,28 @@ public function previewPostNotificationNewsletterInEditor(\AcceptanceTester $i)
     $i->click('Post Notifications', '[data-automation-id="newsletters_listing_tabs"]');
     $i->waitForText($newsletterName);
     $i->clickItemRowActionByItemName($newsletterName, 'Edit');
-    $i->click('[data-automation-id="sidebar_preview_region_heading"]');
+    $i->waitForElement('.mailpoet_show_preview');
+    $i->click('.mailpoet_show_preview');
+    $i->waitForElement('[data-automation-id="switch_send_to_email"]');
+    $i->click('[data-automation-id="switch_send_to_email"]');
     //test sending preview to email
     $i->waitForText('Send preview');
     $i->click('Send preview');
     $i->waitForText('Your test email has been sent!');
     $i->click('.notice-dismiss');
     //check for error if no email is set
+    $i->waitForElement('.mailpoet_show_preview');
+    $i->click('.mailpoet_show_preview');
+    $i->waitForElement('[data-automation-id="switch_send_to_email"]');
+    $i->click('[data-automation-id="switch_send_to_email"]');
     $i->clearField('#mailpoet_preview_to_email');
     $i->click('Send preview');
     $i->waitForText('Enter an email address to send the preview newsletter to.');
     //set different email and test sending
     $i->fillField('#mailpoet_preview_to_email', 'test2@test.com');
     $i->click('Send preview');
     $i->waitForText('Your test email has been sent!');
-    //check the preview email in browser
```

Comment:
Why was this test section removed?

Reply:
Because it doesn't make sense any more. The button `View in browser` has been removed and the preview modal works - we opened it a couple of lines above.